pkg/provider-local/imagevector/imagevector.go

```diff
@@ -2,7 +2,7 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-//go:generate ../../../hack/generate-imagename-constants.sh
+//go:generate ../../../hack/generate-imagename-constants.sh imagevector images.yaml
```

Comment:
/cc @plkokanov as a heads-up for https://github.com/gardener/gardener-extension-shoot-rsyslog-relp/blob/d91f6f72c094eaf8a7f312c5916a7ccbcfc9e5bc/imagevector/imagevector.go#L5.

Reply:
Hm, nothing should change for extensions, the script should be backwards-compatible. Have you noticed an issue?

Reply:
Didn't the default change from `images.yaml` to `containers.yaml` and the extensions need the same adjustment as `provider-local`, if they use the `generate-imagename-constants.sh` script?

Reply:
This was not the intention... I adapted the changes to keep the script backwards-compatible for now.